examples/js/loaders/deprecated/LegacyGLTFLoader.js

```diff
@@ -714,11 +714,15 @@ THREE.LegacyGLTFLoader = ( function () {
 
 			switch ( semantic ) {
 
+				case "POSITION_0":
+				case "POSITION0":
```

Comment:
`POSITION0` syntax is not permitted by the glTF spec:

> Valid attribute semantic property names include POSITION, NORMAL, TEXCOORD, COLOR, JOINT, and WEIGHT. Attribute semantic property names can be of the form [semantic]_[set_index], e.g., TEXCOORD_0, TEXCOORD_1, etc.

So I don't think that case needs to be added here, similarly below.

Reply:
I'll submit yet another version that leaves this section alone entirely.

Reply:
By the way, you can also just make new commits on your branch and `git push`, they'll appear in this PR automatically. If it's easier to start a new branch and PR that's fine too, though.

Reply:
Oh, that's good to know.